lib/src/rules/prefer_null_aware_operators.dart

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/token.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:analyzer/dart/element/element.dart';
+import 'package:linter/src/analyzer.dart';
+
+const _desc = r'Prefer using null aware operator.';
+
+const _details = r'''
+
+Prefer using null aware operator instead of null check in conditional
```

Comment:
=> Prefer using null aware operators instead of null checks in ...

Reply:
oops I missed the plural